breathe/directives.py

```diff
@@ -494,11 +492,7 @@ class DoxygenVariableDirective(DoxygenBaseItemDirective):
     kind = "variable"
 
     def render(self, node_stack, project_info, options, filter_, target_handler, mask_factory):
-        # Remove 'extern' keyword as Sphinx doesn't support it.
         definition = node_stack[0].definition
-        extern = 'extern '
-        if definition.startswith(extern):
-            definition = definition[len(extern):]
```

Comment:
Sphinx 1.4 doesn't seem to support `extern` yet:

```
WARNING: Invalid definition: Expected identifier in nested name, got keyword: extern [error at 6]
  extern int foo
  ------^
```
Doxygen version 1.8.6 doesn't put `extern` in the `definition` field, but IIRC some other version do. Therefore I suggest keeping this.